net/Socket.hpp

```diff
@@ -89,9 +96,16 @@ class SocketDisposition final
     }
     bool isMove() const { return _disposition == Type::MOVE; }
     bool isClosed() const { return _disposition == Type::CLOSED; }
+    bool isTransfer() const { return  _disposition == Type::TRANSFER; }
+    bool isContinue() const { return _disposition == Type::CONTINUE; }
 
     /// Perform the queued up work.
     void execute();
+
+    Type _disposition;
+    MoveFunction _socketMove;
+    SocketPoll *_toPoll;
+    std::shared_ptr<Socket> _socket;
```

Comment:
Do we really need all these publicly visible?

Reply:
Thanks for the input; having a reference to the socketpoll is somewhat inevitable I think without making that a parent class of DocumentBroker, will we could make it a private parent I suppose.